docs/api.md

```diff
@@ -0,0 +1,72 @@
+# Vivify's API
+
+In addition to the command line interface `viv`, Vivify has HTTP endpoints that
```

Comment:
Now, this might be silly, so you can choose to adopt this or not, but I would propose that the project name always be wrapped in bold tags, so that **Vivify** would always appear as bold, as like a branding thing.

```suggestion
In addition to the command line interface `viv`, **Vivify** has HTTP endpoints that
```

Reply:
Hm, I see the point but I feel like that might be a bit cumbersome to do because we would have to do it in all other files as well and will probably forget some, especially as the documentation evolves. I would say the capitalization takes care of this to a good enough level since most other words are lower case in English. What do you think?

Reply:
Yeah that works fine. That's certainly easier.